webpack/scenes/ContentViews/Details/Versions/VersionDetails/ContentViewVersionDetailConfig.js

```diff
@@ -0,0 +1,299 @@
+  selectCVVersions,
+  selectCVVersionsStatus,
+  selectDebPackages,
+  selectDebPackagesStatus,
+  selectDockerTags,
+  selectDockerTagsStatus,
+  selectErrata,
+  selectErrataStatus,
+  selectModuleStreams,
+  selectModuleStreamsStatus,
+  selectRepositories,
+  selectRepositoriesStatus,
+  selectRPMPackageGroups,
+  selectRPMPackageGroupsStatus,
+  selectRPMPackages,
+  selectRPMPackagesStatus,
+} from '../../ContentViewDetailSelectors';
+import ContentViewVersionRepositoryCell from './ContentViewVersionRepositoryCell';
+
+export const TableType = PropTypes.shape({
+  name: PropTypes.string,
+  getCountKey: PropTypes.func,
+  responseSelector: PropTypes.func,
+  statusSelector: PropTypes.func,
+  autocompleteEndpoint: PropTypes.string,
+  fetchItems: PropTypes.func,
+  columnHeaders:
+    PropTypes.arrayOf(PropTypes.shape({
+      title: PropTypes.string,
+      width: PropTypes.number,
+      getProperty: PropTypes.func,
+    })),
+});
+
+
+export default ({ cvId, versionId }) => [
+  {
+    name: __('Components'),
+    getCountKey: item => item?.component_view_count,
+    responseSelector: state => selectCVVersions(state, cvId),
+    statusSelector: state => selectCVVersionsStatus(state, cvId),
+    autocompleteEndpoint: '',
+    fetchItems: params => getContentViewVersions(
+      cvId,
+      { composite_version_id: versionId, ...params, content_view_id: undefined },
+    ),
+    columnHeaders: [
+      {
+        title: __('Content View Name'),
+        getProperty: item => (
+          <a href={urlBuilder(`labs/content_views/${item?.content_view_id}`, '')}>
+            {item?.content_view?.name}
+          </a>),
+      },
+      { title: __('Version'), getProperty: item => item?.version },
+      {
+        title: __('Updated'),
+        getProperty: item => item?.updated_at &&
+          <LongDateTime date={item.updated_at} showRelativeTimeTooltip />,
+      },
+    ],
+    disableSearch: true,
+  },
+  {
+    name: __('Repositories'),
+    getCountKey: item => item?.repositories?.length,
+    responseSelector: state => selectRepositories(state),
+    statusSelector: state => selectRepositoriesStatus(state),
+    autocompleteEndpoint: `/repositories/auto_complete_search?archived=true&content_view_version_id=${versionId}`,
+    fetchItems: params => getRepositories({
+      content_view_version_id: versionId,
+      archived: true,
+      ...params,
+    }),
+    columnHeaders: [
+      {
+        title: __('Name'),
+        getProperty: item => (
+          <a href={urlBuilder(`products/${item?.product?.id}/repositories/${item?.id}`, '')}>
+            {item?.name}
+          </a>),
+      },
+      {
+        title: __('Product'),
+        getProperty: item => (
+          <a href={urlBuilder(`products/${item?.product?.id}`, '')}>
+            {item?.product?.name}
+          </a>),
+      },
+      {
+        title: __('Content'),
+        getProperty: item => <ContentViewVersionRepositoryCell data={item} />,
+      },
+    ],
```

Comment:
A repository type column here will be helpful.

Reply:
I should have thought of that, great catch.